### DISCUSSION TITLE
Mobile score is too low

### DISCUSSION THREAD

**Discussion by Mister-Hope** [Mereithhh/vanblog#170] [Q&A]
![image](https://user-images.githubusercontent.com/33315834/221762996-94569003-b81f-4f0d-a0f2-fb5dfe8451fd.png)
![image](https://user-images.githubusercontent.com/33315834/221763005-a2c36267-338b-4b98-8c34-ff591b0a26c8.png)

Any ideas?

**Comment by Mister-Hope** [comment]
Here is comparation between demo and vuejs.press (using vuepress-theme-hope):

![image](https://user-images.githubusercontent.com/33315834/221763866-bb888e5b-a941-43cd-ae31-12b9c8edc57f.png)
![image](https://user-images.githubusercontent.com/33315834/221763915-3d994595-3f69-4975-b42b-16908286d054.png)


![image](https://user-images.githubusercontent.com/33315834/221763992-c9046d74-4707-4a62-9fce-1a2ae628fad5.png)
![image](https://user-images.githubusercontent.com/33315834/221764022-b14c9d5b-11f1-40da-81dd-4ad0eaf2832f.png)


You can test this by your self at:

- https://pagespeed.web.dev/report?url=https%3A%2F%2Fvuejs.press%2F&form_factor=mobile
- https://pagespeed.web.dev/report?url=https%3A%2F%2Fblog-demo.mereith.com%2F&form_factor=mobile


As a member of VuePress team, VuePress is slow due to its arch (much slower than VitePress and Astro), so I think at least vanblog should beat it before declaring "great performance".

BTW, the pagespeed insights server is not located at china mainland, if your demo server is in china, then you may use score data from browser.

**Reply by Mereithhh** [reply]
You're right.  With the development of function, I ignored the performance optimization. I will work hard to optimize performance, or I will remove the description related to high performance.

PS: All server is in china, I tried in my browser, the score is higher ,but still not so high enough. :)
